Sources/Public/SDK/ATTNSDK.swift

```diff
@@ -147,6 +147,14 @@ public final class ATTNSDK: NSObject {
   public func clearUser() {
     userIdentity.clearUser()
   }
+
+  @objc(updateDomain:)
+  public func update(domain: String) {
```

Comment:
Steps to update domain:
- The new domain **should not** be the same as the previous one
- Update SDK domain property
- Update API domain property
  - cachedGeoAdjustedDomain reset to nil to request the geo adjusted service again
- Trigger identity event with cached data

Reply:
In this case "should not" means it'll just noop, right? Not that we'll see some sort of error.

Reply:
That's correct. The user will not receive errors. The execution will only stop if the new domain is the same as the one already configured.